content/docs/extensions/model/sklearn.md

```diff
@@ -0,0 +1,69 @@
+# Scikit-Learn Models Support
+
+[ModelType](/doc/user-guide/mlem-abcs#modeltype) implementations for any
+sklearn-compatible classes as well as `Pipeline`
+
+## Description
+
+**TODO**
```

Comment:
Haven't seen any descriptions yet. Any ideas for what will be in these sections? Maybe leave it for the user guide if it's unclear now.